renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -386,8 +386,15 @@ ReplayStatus GLInitParams::Serialise()
   return ReplayStatus::Succeeded;
 }
 
-void WrappedOpenGL::GetGLExtensions()
+const vector<string> &WrappedOpenGL::GetGLExtensions()
 {
+  static vector<string> globalExts;
```

Comment:
Oh.. yeah the threads...

I'm fine having two lists, thanks. Will update the PR.